src/main.js

```diff
@@ -289,7 +299,6 @@ class MiniMediaPlayer extends LitElement {
       '--initial': this.initial,
       '--bg': config.background,
       '--group': config.group,
-      '--more-info': config.more_info,
```

Comment:
This was used to determine if we should show a mouse pointer or the default cursor, we should keep this but instead  `config.tap_action !== 'none'`.